WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -61,8 +68,9 @@ final class OrdersRootViewController: UIViewController {
         self.siteID = siteID
         super.init(nibName: Self.nibName, bundle: nil)
 
+        // workaround to get rid of the extra space at the bottom when embedded in split view
+        extendedLayoutIncludesOpaqueBars = true
```

Comment:
🤭 I think optimizing for big screens will give us a big headache. It is full of small details that are easy to forget.

Reply:
I believe we use opaque bars somewhere in the app (the navigation bars maybe), which causes the need for this. The default behavior for bars since iOS 7 is translucent, and this property only needs to be set to true if we use opaque bars.

Reply:
I can confirm that this only happens with views embedded in a navigation controller that is added to a split view. Since we're using opaque navigation bars, `extendedLayoutIncludesOpaqueBars` needs to be true to get rid of the extra space at the bottom.

Reply:
Come to think of it, maybe we should make sure that all child view controllers of our custom navigation bar has `extendedLayoutIncludesOpaqueBars` set to true. Updated this in 8a6cc081f63f95531de7edc63466ae8b792924b7.